pkg/operator/ceph/config/monstore_test.go

```diff
@@ -72,6 +73,89 @@ func TestMonStore_Set(t *testing.T) {
 	assert.Contains(t, execedCmd, " config set mon.* unknown_setting 10 ")
 }
 
+func TestMonStore_Delete(t *testing.T) {
+	executor := &exectest.MockExecutor{}
+	clientset := testop.New(t, 1)
+	ctx := &clusterd.Context{
+		Clientset: clientset,
+		Executor:  executor,
+	}
+
+	// create a mock command runner which creates a simple string of the command it ran, and allow
+	// us to cause it to return an error when it detects a keyword.
+	execedCmd := ""
+	execInjectErr := false
+	executor.MockExecuteCommandWithOutputFile =
+		func(command string, outfile string, args ...string) (string, error) {
+			execedCmd = command + " " + strings.Join(args, " ")
+			if execInjectErr {
+				return "output from cmd with error", errors.New("mocked error")
+			}
+			return "", nil
+		}
+
+	monStore := GetMonStore(ctx, "ns")
+
+	// ceph config rm called as expected
+	e := monStore.Delete("global", "debug ms")
+	assert.NoError(t, e)
+	assert.Contains(t, execedCmd, "config rm global debug_ms")
+
+	// errors returned as expected
+	execInjectErr = true
+	e = monStore.Delete("mon.*", "unknown_setting")
+	assert.Error(t, e)
+	assert.Contains(t, execedCmd, " config rm mon.* unknown_setting ")
+}
+
+func TestMonStore_GetDaemon(t *testing.T) {
```

Comment:
```suggestion
func TestMonStoreGetDaemon(t *testing.T) {
```

Reply:
Same: I prefer to keep the same naming than other test, for ex.: TestMonStore_Set